src/chttpd_cors.erl

```diff
@@ -306,6 +316,35 @@ get_cors_config(#httpd{cors_config = undefined}) ->
 get_cors_config(#httpd{cors_config = Config}) ->
     Config.
 
+cors_config(Host, Key, Default) ->
+    config:get(cors_section(Host), Key,
+                     config:get("cors", Key, Default)).
+
+cors_section(Host0) ->
+    {Host, _Port} = split_host_port(Host0),
```

Comment:
I don't get it, why not just `string:tokens/2` here? Or after `mochiweb_util:urlsplit/1` if `couch_httpd_vhost:host/1` is returning host with scheme.

Speaking of, vhost's cors config in default.ini mentions that scheme must to be included, are we dropping this req?